sdk/turing/batch/job.py

```diff
@@ -145,6 +145,7 @@ def list(
         status: List[EnsemblingJobStatus] = None,
         page: Optional[int] = None,
         page_size: Optional[int] = None,
+        ensembler_id: Optional[int] = None,
```

Comment:
Why is this added to the signature? Does this even work? If I remembered correctly, the API `ListEnsemblingJobs` endpoint does not support the `ensembler_id` as a query param right?

Reply:
Yes this is working, https://github.com/caraml-dev/turing/blob/58f66852ccb6861cda4a4854d6e658ebd72cf8d3/api/turing/service/ensembling_job_service.go#L178-L180
we could search list of jobs by ensembler 😁

Reply:
Awesome! Thanks for confirming!